<!DOCTYPE html>
<html>
  <head>
    <title>OBO Community Health</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta charset="UTF-8">
    <!-- Bootstrap -->
    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link
        rel="stylesheet"
        href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"
    />
    <link
        rel="stylesheet"
        href="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.css"
    />

    <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
    <link
        rel="stylesheet"
        href="https://cdn.datatables.net/1.10.19/css/dataTables.bootstrap4.min.css"
    />
    <style type="text/css">
        h1, h2 {
            margin-top: 35px;
        }
    </style>
  </head>
  <body data-spy="scroll" data-target="#toc">
    
    
<div class="container">
    <h1 align="center">OBO Community Health</h1>
    <p>
        This page gives insight into which people are responsible for which OBO Foundry ontologies,
        how easy they are to get in contact with, and what the extent of their responsibilities across
        the OBO Foundry are. It was generated using the GitHub API with scripts at
        <a href="https://github.com/cthoyt/obo-community-health">https://github.com/cthoyt/obo-community-health</a>.
    </p>

    <table class="table table-striped table-hover" id="table-contacts">
        <thead>
        <tr style="text-align: right;">
            <th>#</th>
            <th>Name</th>
            <th>GitHub</th>
            
            <th>Ontologies</th>
        </tr>
        </thead>
        <tbody>
        
        <tr>
            <td align="right">4</td>
            <td><a href="mailto:jiezheng@pennmedicine.upenn.edu">Jie Zheng</a></td>
            <td><a href="https://github.com/zhengj2007">zhengj2007</a></td>
            
            <td><a href="https://obofoundry.org/ontology/bcgo" data-toggle="tooltip" data-html="true"  title="<strong>Beta Cell Genomics Ontology</strong><br />An application ontology built for beta cell genomics studies.">bcgo</a>,  <a href="https://obofoundry.org/ontology/iao" data-toggle="tooltip" data-html="true"  title="<strong>Information Artifact Ontology</strong><br />An ontology of information entities, originally driven by work by the Ontology of Biomedical Investigation (OBI) digital entity and realizable information entity branch.">iao</a>,  <a href="https://obofoundry.org/ontology/obcs" data-toggle="tooltip" data-html="true"  title="<strong>Ontology of Biological and Clinical Statistics</strong><br />OBCS stands for the Ontology of Biological and Clinical Statistics. OBCS is an ontology in the domain of biological and clinical statistics. It is aligned with the Basic Formal Ontology (BFO) and the Ontology for Biomedical Investigations (OBI). OBCS imports all possible biostatistics terms in OBI and includes many additional biostatistics terms, some of which were proposed and discussed in the OBI face-to-face workshop in Ann Arbor in 2012.  ">obcs</a>,  <a href="https://obofoundry.org/ontology/obib" data-toggle="tooltip" data-html="true"  title="<strong>Ontology for Biobanking</strong><br />The Ontology for Biobanking (OBIB) is an ontology for the annotation and modeling of the activities, contents, and administration of a biobank. Biobanks are facilities that store specimens, such as bodily fluids and tissues, typically along with specimen annotation and clinical data. OBIB is based on a subset of the Ontology for Biomedical Investigation (OBI), has the Basic Formal Ontology (BFO) as its upper ontology, and is developed following OBO Foundry principles. The first version of OBIB resulted from the merging of two existing biobank-related ontologies, OMIABIS and biobank ontology.">obib</a> </td>
        </tr>
        
        <tr>
            <td align="right">3</td>
            <td><a href="mailto:jrsmith@mcw.edu">Jennifer Smith</a></td>
            <td><a href="https://github.com/jrsjrs">jrsjrs</a></td>
            
            <td><a href="https://obofoundry.org/ontology/cmo" data-toggle="tooltip" data-html="true"  title="<strong>Clinical measurement ontology</strong><br />Morphological and physiological measurement records generated from clinical and model organism research and health programs.">cmo</a>,  <a href="https://obofoundry.org/ontology/mmo" data-toggle="tooltip" data-html="true"  title="<strong>Measurement method ontology</strong><br />A representation of the variety of methods used to make clinical and phenotype measurements. ">mmo</a>,  <a href="https://obofoundry.org/ontology/xco" data-toggle="tooltip" data-html="true"  title="<strong>Experimental condition ontology</strong><br />Conditions under which physiological and morphological measurements are made both in the clinic and in studies involving humans or model organisms.">xco</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:hescriva@obs-banyuls.fr">Hector Escriva</a></td>
            <td><a href="https://github.com/hescriva">hescriva</a></td>
            
            <td><a href="https://obofoundry.org/ontology/amphx" data-toggle="tooltip" data-html="true"  title="<strong>The Amphioxus Development and Anatomy Ontology</strong><br />An ontology for the development and anatomy of Amphioxus (Branchiostoma lanceolatum).">amphx</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:stacia@stanford.edu">Stacia R Engel</a></td>
            <td><a href="https://github.com/srengel">srengel</a></td>
            
            <td><a href="https://obofoundry.org/ontology/apo" data-toggle="tooltip" data-html="true"  title="<strong>Yeast phenotypes</strong><br />A structured controlled vocabulary for the phenotypes of Ascomycete fungi.">apo</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:amalik@ebi.ac.uk">Adnan Malik</a></td>
            <td><a href="https://github.com/amalik01">amalik01</a></td>
            
            <td><a href="https://obofoundry.org/ontology/chebi" data-toggle="tooltip" data-html="true"  title="<strong>Chemical Entities of Biological Interest</strong><br />Chemical Entities of Biological Interest (ChEBI) is a freely available dictionary of molecular entities focused on &#39;small&#39; chemical compounds.">chebi</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:siiraa@umich.edu">Sirarat Sarntivijai</a></td>
            <td><a href="https://github.com/siiraa">siiraa</a></td>
            
            <td><a href="https://obofoundry.org/ontology/clo" data-toggle="tooltip" data-html="true"  title="<strong>Cell Line Ontology</strong><br />The Cell Line Ontology (CLO) is a community-based ontology of cell lines. The CLO is developed to unify publicly available cell line entry data from multiple sources to a standardized logically defined format based on consensus design patterns.">clo</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:whimar@ohsu.edu">Marijane White</a></td>
            <td><a href="https://github.com/marijane">marijane</a></td>
            
            <td><a href="https://obofoundry.org/ontology/cro" data-toggle="tooltip" data-html="true"  title="<strong>Contributor Role Ontology</strong><br />A classification of the diverse roles performed in the work leading to a published research output in the sciences. Its purpose to provide transparency in contributions to scholarly published work, to enable improved systems of attribution, credit, and accountability.">cro</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:mcourtot@gmail.com">Melanie Courtot</a></td>
            <td><a href="https://github.com/mcourtot">mcourtot</a></td>
            
            <td><a href="https://obofoundry.org/ontology/duo" data-toggle="tooltip" data-html="true"  title="<strong>Data Use Ontology</strong><br />DUO is an ontology which represent data use conditions.">duo</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:mgiglio@som.umaryland.edu">Michelle Giglio</a></td>
            <td><a href="https://github.com/mgiglio99">mgiglio99</a></td>
            
            <td><a href="https://obofoundry.org/ontology/eco" data-toggle="tooltip" data-html="true"  title="<strong>Evidence ontology</strong><br />Evidence codes can be used to specify the type of supporting evidence for a piece of knowledge. This allows inference of a &#39;level of support&#39; between an entity and an annotation made to an entity.">eco</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:georgeta.bordea@u-bordeaux.fr">Georgeta Bordea</a></td>
            <td><a href="https://github.com/getbordea">getbordea</a></td>
            
            <td><a href="https://obofoundry.org/ontology/fideo" data-toggle="tooltip" data-html="true"  title="<strong>Food Interactions with Drugs Evidence Ontology</strong><br />The Food Interactions with Drugs Evidence Ontology (FIDEO) represents Food-Drug Interactions and underlying interaction mechanisms described in scientific publications, drug and adverse effects databases, and drug interactions compendia. The ontology builds on previous efforts from the FoodOn, DRON, ChEBI, and DIDEO ontologies as well as the Thériaque database. This ontology is maintained at https://gitub.u-bordeaux.fr/erias/fideo, and requests for changes or additions should be filed at the issue tracker there.">fideo</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:polcaes@gmail.com">Pol Castellano Escuder</a></td>
            <td><a href="https://github.com/pcastellanoescuder">pcastellanoescuder</a></td>
            
            <td><a href="https://obofoundry.org/ontology/fobi" data-toggle="tooltip" data-html="true"  title="<strong>Food-Biomarker Ontology</strong><br />FOBI (Food-Biomarker Ontology) is an ontology to represent food intake data and associate it with metabolomic data">fobi</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:rbca.jackson@gmail.com">Rebecca Jackson</a></td>
            <td><a href="https://github.com/beckyjackson">beckyjackson</a></td>
            
            <td><a href="https://obofoundry.org/ontology/gecko" data-toggle="tooltip" data-html="true"  title="<strong>Genomics Cohorts Knowledge Ontology</strong><br />An ontology to represent genomics cohort attributes.">gecko</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:nje5@georgetown.edu">Nathan Edwards</a></td>
            <td><a href="https://github.com/edwardsnj">edwardsnj</a></td>
            
            <td><a href="https://obofoundry.org/ontology/gno" data-toggle="tooltip" data-html="true"  title="<strong>Glycan Naming and Subsumption Ontology</strong><br />An ontology for glycans based on GlyTouCan, but organized by subsumption.">gno</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:aellenhicks@gmail.com">Amanda Hicks</a></td>
            <td><a href="https://github.com/aellenhicks">aellenhicks</a></td>
            
            <td><a href="https://obofoundry.org/ontology/htn" data-toggle="tooltip" data-html="true"  title="<strong>Hypertension Ontology</strong><br />An ontology for representing clinical data about hypertension, intended to support classification of patients according to various diagnostic guidelines">htn</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:liumeng94@sjtu.edu.cn">Meng LIU</a></td>
            <td><a href="https://github.com/Lemon-Liu">Lemon-Liu</a></td>
            
            <td><a href="https://obofoundry.org/ontology/iceo" data-toggle="tooltip" data-html="true"  title="<strong>Integrative and Conjugative Element Ontology</strong><br />A biological ontology to standardize and integrate Integrative and Conjugative Element (ICE) information and to support computer-assisted reasoning.">iceo</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:druzinsk@uic.edu">Robert Druzinsky</a></td>
            <td><a href="https://github.com/RDruzinsky">RDruzinsky</a></td>
            
            <td><a href="https://obofoundry.org/ontology/mfmo" data-toggle="tooltip" data-html="true"  title="<strong>Mammalian Feeding Muscle Ontology</strong><br />The Mammalian Feeding Muscle Ontology is an antomy ontology for the muscles of the head and neck that participate in feeding, swallowing, and other oral-pharyngeal behaviors.">mfmo</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:pporras@ebi.ac.uk">Pablo Porras Millán</a></td>
            <td><a href="https://github.com/pporrasebi">pporrasebi</a></td>
            
            <td><a href="https://obofoundry.org/ontology/mi" data-toggle="tooltip" data-html="true"  title="<strong>Molecular Interactions Controlled Vocabulary</strong><br />The Molecular Interactions (MI) ontology forms a structured controlled vocabulary for the annotation of experiments concerned with protein-protein interactions. MI is developed by the HUPO Proteomics Standards Initiative.">mi</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:hilmar.lapp@duke.edu">Hilmar Lapp</a></td>
            <td><a href="https://github.com/hlapp">hlapp</a></td>
            
            <td><a href="https://obofoundry.org/ontology/miapa" data-toggle="tooltip" data-html="true"  title="<strong>Minimum Anformation About a Phylogenetic Analysis Ontology</strong><br />The MIAPA ontology is intended to be an application ontology for the purpose of semantic annotation of phylogenetic data according to the requirements and recommendations of the Minimum Information for A Phylogenetic Analysis (MIAPA) metadata reporting standard. The ontology leverages (imports) primarily from the CDAO (Comparative Data Analysis Ontology), PROV (W3C Provenance Ontology), and SWO (Software Ontology, which includes the EDAM ontologies) ontologies. It adds some assertions of its own, as well as some classes and individuals that may eventually get pushed down into one of the respective source ontologies.

This ontology is maintained at http://github.com/miapa/miapa, and requests for changes or additions should be filed at the issue tracker there. The discussion list is at miapa-discuss@googlegroups.com. Further resources about MIAPA can be found at the project&#39;s main page at http://evoio.org/wiki/MIAPA.">miapa</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:pierre-alain.binz@chuv.ch">Pierre-Alain Binz</a></td>
            <td><a href="https://github.com/pabinz">pabinz</a></td>
            
            <td><a href="https://obofoundry.org/ontology/mod" data-toggle="tooltip" data-html="true"  title="<strong>Protein modification</strong><br />The Proteomics Standards Initiative modification ontology (PSI-MOD)  aims to define a concensus nomenclature and ontology reconciling, in a hierarchical representation, the complementary descriptions of residue modifications.">mod</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:pns12@hermes.cam.ac.uk">Paul Schofield</a></td>
            <td><a href="https://github.com/PaulNSchofield">PaulNSchofield</a></td>
            
            <td><a href="https://obofoundry.org/ontology/mpath" data-toggle="tooltip" data-html="true"  title="<strong>Mouse pathology ontology</strong><br />A structured controlled vocabulary of mutant and transgenic mouse pathology phenotypes">mpath</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:chen.yang@ugent.be">Chen Yang</a></td>
            <td><a href="https://github.com/cyang0128">cyang0128</a></td>
            
            <td><a href="https://obofoundry.org/ontology/one" data-toggle="tooltip" data-html="true"  title="<strong>Ontology for Nutritional Epidemiology</strong><br />An ontology to standardize research output of nutritional epidemiologic studies.">one</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:francesco.vitali@ibba.cnr.it">Francesco Vitali</a></td>
            <td><a href="https://github.com/FrancescoVit">FrancescoVit</a></td>
            
            <td><a href="https://obofoundry.org/ontology/ons" data-toggle="tooltip" data-html="true"  title="<strong>Ontology for Nutritional Studies</strong><br />The Ontology for Nutritional Studies (ONS) has been developed as part of the ENPADASI European project (http://www.enpadasi.eu/) with the aim to define a common language and building ontologies for nutritional studies.">ons</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:fortuna@ebd.csic.es">Miguel A. Fortuna</a></td>
            <td><a href="https://github.com/miguelfortuna">miguelfortuna</a></td>
            
            <td><a href="https://obofoundry.org/ontology/ontoavida" data-toggle="tooltip" data-html="true"  title="<strong>Ontology for Avida digital evolution platform</strong><br />The Ontology for Avida (OntoAvida) project aims to develop an integrated vocabulary for the description of the most widely used computational approach for performing experimental evolution using digital organisms (i.e., self-replicating computer programs that evolve within a user-defined computational environment).

The lack of a clearly defined vocabulary makes biologists feel reluctant to embrace the field of digital evolution. This unique ontology has the potential to change this picture overnight.

OntoAvida was initially developed by https://fortunalab.org, the computational biology lab at the Doñana Biological Station (a research institute of the Spanish National Research Council based at Seville, Spain). Contributors to OntoAvida are expected to include members of the Digital Evolution Laboratory (https://devolab.org/) at Michigan State University (USA).

More information can be found at https://obofoundry.org/ontology/ontoavida.html">ontoavida</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:fernanda.farinelli@gmail.com">Fernanda Farinelli</a></td>
            <td><a href="https://github.com/FernandaFarinelli">FernandaFarinelli</a></td>
            
            <td><a href="https://obofoundry.org/ontology/ontoneo" data-toggle="tooltip" data-html="true"  title="<strong>Obstetric and Neonatal Ontology</strong><br />The Obstetric and Neonatal Ontology is a structured controlled vocabulary to provide a representation of the data from electronic health records (EHRs) involved in the care of the pregnant woman, and of her baby.">ontoneo</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:safisher@upenn.edu">Stephen Fisher</a></td>
            <td><a href="https://github.com/safisher">safisher</a></td>
            
            <td><a href="https://obofoundry.org/ontology/ornaseq" data-toggle="tooltip" data-html="true"  title="<strong>Ontology of RNA Sequencing</strong><br />An application ontology designed to annotate next-generation sequencing experiments performed on RNA.">ornaseq</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:gthayman@mcw.edu">G. Thomas Hayman</a></td>
            <td><a href="https://github.com/gthayman">gthayman</a></td>
            
            <td><a href="https://obofoundry.org/ontology/pw" data-toggle="tooltip" data-html="true"  title="<strong>Pathway ontology</strong><br />The Pathway Ontology captures information on biological networks, the relationships between netweorks and the alterations or malfunctioning of such networks within a hierarchical structure. The five main branches of the ontology are: classic metabolic pathways, regulatory, signaling, drug, and disease pathwaysfor complex human conditions.">pw</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:sjwang@mcw.edu">Shur-Jen Wang</a></td>
            <td><a href="https://github.com/shurjenw">shurjenw</a></td>
            
            <td><a href="https://obofoundry.org/ontology/rs" data-toggle="tooltip" data-html="true"  title="<strong>Rat Strain Ontology</strong><br />Ontology of rat strains">rs</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:sheriff@ebi.ac.uk">Rahuman Sheriff</a></td>
            <td><a href="https://github.com/rsmsheriff">rsmsheriff</a></td>
            
            <td><a href="https://obofoundry.org/ontology/sbo" data-toggle="tooltip" data-html="true"  title="<strong>Systems Biology Ontology</strong><br />The goal of the Systems Biology Ontology is to develop controlled vocabularies and ontologies tailored specifically for the kinds of problems being faced in Systems Biology, especially in the context of computational modeling. SBO is a project of the BioModels.net effort.">sbo</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:giant.plankton@gmail.com">Jade Hotchkiss</a></td>
            <td><a href="https://github.com/JadeHotchkiss">JadeHotchkiss</a></td>
            
            <td><a href="https://obofoundry.org/ontology/scdo" data-toggle="tooltip" data-html="true"  title="<strong>Sickle Cell Disease Ontology</strong><br />An ontology for the standardization of terminology and integration of knowledge about Sickle Cell Disease.">scdo</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:ramirez@macn.gov.ar">Martin Ramirez</a></td>
            <td><a href="https://github.com/martinjramirez">martinjramirez</a></td>
            
            <td><a href="https://obofoundry.org/ontology/spd" data-toggle="tooltip" data-html="true"  title="<strong>Spider Ontology</strong><br />An ontology for spider comparative biology including anatomical parts (e.g. leg, claw), behavior (e.g. courtship, combing) and products (i.g. silk, web, borrow).">spd</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:y-yamagata@nibiohn.go.jp">Yuki Yamagata</a></td>
            <td><a href="https://github.com/yuki-yamagata">yuki-yamagata</a></td>
            
            <td><a href="https://obofoundry.org/ontology/txpo" data-toggle="tooltip" data-html="true"  title="<strong>Toxic Process Ontology</strong><br />Elucidating the mechanism of toxicity is crucial in drug safety evaluations. TOXic Process Ontology (TXPO) systematizes a wide variety of terms involving toxicity courses and processes. The first version of TXPO focuses on liver toxicity.

The TXPO contains an is-a hierarchy that is organized into three layers: the top layer contains general terms, mostly derived from the Basic Formal Ontology. The intermediate layer contains biomedical terms in OBO foundry from UBERON, Cell Ontology, NCBI Taxon, ChEBI, Gene Ontology, PATO, OGG, INOH, HINO, NCIT, DOID and Relational ontology (RO). The lower layer contains toxicological terms.

In applied work, we have developed a prototype of TOXPILOT, a TOXic Process InterpretabLe knOwledge sysTem. TOXPILOT provides visualization maps of the toxic course, which facilitates capturing the comprehensive picture for understanding toxicity mechanisms. A prototype of TOXPILOT is available: https://toxpilot.nibiohn.go.jp">txpo</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:caripark@iastate.edu">Carissa Park</a></td>
            <td><a href="https://github.com/caripark">caripark</a></td>
            
            <td><a href="https://obofoundry.org/ontology/vt" data-toggle="tooltip" data-html="true"  title="<strong>Vertebrate trait ontology</strong><br />An ontology of traits covering vertebrates">vt</a> </td>
        </tr>
        
        <tr>
            <td align="right">1</td>
            <td><a href="mailto:raymond@caltech.edu">Raymond Lee</a></td>
            <td><a href="https://github.com/raymond91125">raymond91125</a></td>
            
            <td><a href="https://obofoundry.org/ontology/wbbt" data-toggle="tooltip" data-html="true"  title="<strong>C. elegans Gross Anatomy Ontology</strong><br />Ontology about the gross anatomy of the C. elegans">wbbt</a> </td>
        </tr>
        
        </tbody>
    </table>
</div>


    

    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
    <script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.js"></script>

    <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
    <script src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.min.js"></script>
    <script src="https://cdn.datatables.net/1.10.19/js/dataTables.bootstrap4.min.js"></script>

    <script src="https://d3js.org/d3.v5.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.js"></script>
<script>
    $(document).ready(function () {
        $("#table-contacts").DataTable({
            "order": [[0, "desc"], ["1", "asc"]],
            "scrollX": true
        });
    })
    $(function () {
        $('[data-toggle="tooltip"]').tooltip()
    })
</script>

  </body>
</html>